backend/server.py

```diff
@@ -63,17 +76,41 @@ async def upload(file: UploadFile = Form(...),
         # Delete the temporary chunks directory
         shutil.rmtree(CHUNKS_DIR.parent)
         
-        # STT Process
+        # STT Process   
+        print('started STT proccess...')
```

Comment:
can you extract the whole stt phase in general to another method that will sit in another class? the `server.py` should not contain so many unrelated stuff